sources/packages/web/src/components/common/OverawardDetails.vue

```diff
@@ -187,7 +193,7 @@ export default defineComponent({
       if (manualOveraward) {
         try {
           await OverawardService.shared.addManualOverawardDeduction(
-            props.studentId,
+            props.studentId as number,
```

Comment:
Is it not the case of `addManualOverawardDeduction` having the `studentId` as optional?

Reply:
As we talked about, because it is only for the Ministry and the method is shared between student/ministry, the `studentId` should be changed to be optional on the method `addManualOverawardDeduction`.

Reply:
The point here is that `addManualOverawardDeduction` will be called only by the Ministry and therefore `studentId` is mandatory, right?